README.md

```diff
@@ -569,6 +565,24 @@ We will also see this when working with scripts in VS Code under the problems pa
 
 ![image](https://user-images.githubusercontent.com/1034073/65727087-0e982600-e0b7-11e9-8fa0-d16331ab948a.png)
 
+## Specifying an SDK
+
+Starting with `dotnet-script` 1.4.0 we can now specify the SDK to be used for a script. 
+
+For instance, creating a web server in a script is now as simple as the following.
+
+```csharp
+#r "sdk:Microsoft.NET.Sdk.Web"
+
+using Microsoft.AspNetCore.Builder;
+
+var a = WebApplication.Create();
+a.MapGet("/", () => "Hello world");
+a.Run();
+```
+
+> Please note the the only SDK currently supported is `Microsoft.NET.Sdk.Web`
```

Comment:
different project SDKs have [different default namespaces](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/overview#implicit-using-directives) which we do not currently do with this implementation. 

we should either have a code path that inserts different default usings based on SDK, or (which I think is fine) we can add as a note here too that this is also not supported for now

Reply:
Yeah, I actually started on this one, but realized that we need to align this with OmniSharp. One way to solve this is to pass the usings to OmniSharp as part of the compilation dependencies. I think we will save that for another PR as this is getting big enough as it is. Creating a comment on this in the code